code/modules/client/preferences.dm

```diff
@@ -9,7 +9,7 @@ var/global/list/special_roles = list(
 	ROLE_CHANGELING   = IS_MODE_COMPILED("changeling"),
 	ROLE_CULTIST      = IS_MODE_COMPILED("cult"),
 	ROLE_PLANT        = 1,
-	"infested monkey" = IS_MODE_COMPILED("monkey"),
+//	"infested monkey" = IS_MODE_COMPILED("monkey"),
```

Comment:
I call this a bugfix, because its broken in an unclear way. If you try to use the menu with this entry in it it draws a big fat error. I also tried integrating it into the menu and it simply didn't function properly.